src/main/java/com/powsybl/openloadflow/sensi/WoodburyEngine.java

```diff
@@ -105,22 +193,26 @@ public DenseMatrix run(DenseMatrix preContingencyStates) {
     }
 
     /**
-     * Calculate post-contingency states values using pre-contingency states values and some flow transfer factors (alphas).
-     * @return an array of post-contingency voltage angle states (which is the same modified pre-contingency java object).
+     * Calculate post-contingency and post-actions states values, using pre-contingency states values and some flow transfer factors (alphas).
+     * @return an array of post-contingency and post-actions voltage angle states (which is the same modified pre-contingency java object).
      */
     public double[] run(double[] preContingencyStates) {
         Objects.requireNonNull(preContingencyStates);
-        double[] postContingencyStates = preContingencyStates;
+        double[] postContingencyAndOperatorStrategyStates = preContingencyStates;
         setAlphas(new DenseMatrix(preContingencyStates.length, 1, preContingencyStates), 0);
```

Comment:
Please make sure that this frequent allocation of a new DenseMatrix will not create memory issues on large networks with a large number contingencies (that would create many allocations), as we have seen for sensitivity analysis.

Reply:
Closing this one. Discussion to be reopen if we see memory issues in large models.